src/hardware/serialport/serialport.cpp

```diff
@@ -1343,6 +1345,15 @@ class SERIALPORTS final : public Module_base {
 				}
 			}
 #endif
+			else if(type=="serialmouse") {
+				serialports[i] = new CSerialMouse (i, &cmd);
+				serialports[i]->serialType = SERIAL_PORT_TYPE::MOUSE;
+				cmd.GetStringRemain(serialports[i]->commandLineString);
+				if (!serialports[i]->InstallationSuccessful)  {
+					delete serialports[i];
+					serialports[i] = NULL;
+				}
+			}
```

Comment:
formatting

Reply:
Used the script to correct all the formatting.